CDCgov/prime-reportstream
# Reorganize transform schemas for the Universal Pipeline

This PR involves moving some schema files around to have the directory structure be more accurate and descriptive of the schemas. It also involved adding some baseline documentation to the repo to cover sender transforms in the UP.

Test Steps:
1. Reload settings
2. Issue requests to receivers with updated settings and from senders with updated settings
3. Verify the resulting messages haven't changed, and errors don't occur.

## Migration Plan
This will affect existing settings in staging and in prod,  so to minimize impact, will update affected senders and receivers on staging immediately after merging the PR, and then on prod as part of the deployment that contains these changes. Will work with Angela who is leading deployments this sprint to make sure things are handled appropriately.

The query to find the affected senders/receivers is:
```
select setting.name as setting_name, setting.type, orgs.name as organization_name,
  coalesce(setting.values->>'schemaName', setting.values#>>'{translation,schemaName}') as schema
from setting left join setting as orgs on setting.organization_id=orgs.setting_id 
where (setting.values->>'schemaName' like '%fhir_transforms%' or
  setting.values#>>'{translation,schemaName}' like '%hl7_mapping%') and setting.is_active
```

I will use the UI to update the resulting settings on staging after merging and on prod after deployment. There were two receivers that need updates in prod, and 6 that need updates in staging. No senders need updates at this point.

## Changes
- Moved existing transform schemas into new folders
- Adjusted existing references to those schemas
- Added a document for UP sender transforms

## Checklist

### Testing

## Linked Issues
- Fixes #9101
